engine-api/src/main/java/io/nosqlbench/engine/api/activityimpl/SimpleActivity.java

```diff
@@ -255,7 +255,7 @@ public Timer getResultTimer() {
 
     @Override
     public void setPhaseLimiter(RateLimiter rateLimiter) {
-        this.phaseLimiter = phaseLimiter;
+        this.phaseLimiter = rateLimiter;
```

Comment:
`setPhaseLimiter` doesnt seem to be used anywhere but this was a no-op

Reply:
Phases were/are being _phased_ out. This was too fine-grained of an abstraction based on from its use in the CQL driver for paging.